cfme/tests/configure/test_start_page.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import fauxfactory
+
+from cfme.base.credential import Credential
+from cfme.configure.settings import Visual
+from cfme.configure.access_control import Group, User
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+faux_value = fauxfactory.gen_alphanumeric()
+pytestmark = pytest.mark.usefixtures('browser')
+
+
+def get_landing_page():
+    view = navigate_to(Visual, 'All')
+    landing_pages = view.visualstartpage.show_at_login.all_options
+    start_page = []
+    for landing_page in landing_pages:
+        start_page.append(landing_page.text)
+    return start_page
+
+
+@pytest.yield_fixture(scope="module")
+def rbac_user(appliance, role):
+    role = role
+    group = Group(description='test{}'.format(faux_value), role=role.name)
+    group.create()
+    group_user = Group('test{}'.format(faux_value))
+    username = 'test{}'.format(faux_value.lower())
+    password = 'test'
+    cred = Credential(principal=username, secret=password)
+    user = User(name='test{}'.format(faux_value),
+                credential=cred, group=group_user)
+    user.create()
+    login_page = navigate_to(appliance.server, 'LoginScreen')
+    login_page.log_in(user)
+    view = navigate_to(Visual, 'All')
+    landing_pages = view.visualstartpage.show_at_login.all_options
+    start_page = []
+    for landing_page in landing_pages:
+        start_page.append(landing_page.text)
+    yield user, landing_pages
+    login_page = navigate_to(appliance.server, 'LoginScreen')
+    login_page.login_admin()
+    user.delete()
+    group.delete()
+
+
+def set_landing_page(value, appliance, user=None):
+    # This list contains the list of pages which show some error or alerts after login.
+    # TODO remove all these pages when BZ is closed.
+    page_list = ['Bottlenecks', 'Automate Log', 'Compute / Containers / Containers']
+    view = navigate_to(Visual, 'All')
+    if (view.visualstartpage.show_at_login.fill(value) and
+            not any(substring in value for substring in page_list)):
+        view.save.click()
+        login_page = navigate_to(appliance.server, 'LoginScreen')
+        if user is None:
+            login_page.login_admin()
+        else:
+            login_page.log_in(user)
+        logged_in_page = navigate_to(appliance.server, 'LoggedIn')
```

Comment:
**Required**
This is logging in as appliance.user, completely negating the `login_admin` or `log_in` calls that you made on line 59 or 61.

This needs to be removed, and you need to create a BaseLoggedInPage view object in order to check if the login was successful.

Please checkout how the LoggedIn step works, and consider reworking the test to actually set appliance.user and just use the navigation destination.